packages/modules/cra-template-desktop-viewer/template/.env

```diff
@@ -0,0 +1,13 @@
+# ---- Configuration ----
+ITWIN_VIEWER_SCOPE ="email rbac-user:external-client projectwise-share urlps-third-party general-purpose-imodeljs-backend product-settings-service context-registry-service:read-only imodelhub organization profile openid imodeljs-router"
```

Comment:
`general-purpose-imodeljs-backend`, `imodeljs-router`, and `urlps-third-party` can be removed.

Reply:
removed